backuprestore.adoc

```diff
@@ -319,10 +319,9 @@ delete old backups when they are no longer needed.
 * `zmbackupquery` -- This command lists the information about ongoing and
 completed backups, including labels and dates.
 * `zmrestore` -- This command executes a full or incremental restore to the
-Zimbra mail server. The *zmrestore* command is performed on a server
-that is running.
-* `zmrestoreoffline` -- This command restores the Zimbra mail server when
-the mailboxd process is stopped.
+Zimbra mail server. Should only be performed on a running mailbox server.
+* `zmrestoreoffline` -- This command restores a full or incremental restore
+ to the Zimbra mail server. Should only be performed on a stopped mailbox server.
```

Comment:
I still find this ambiguous at this point.  Suggest providing a link to excellent later new section with wording "see later for notes on prerequisites for running zmrestoreoffline"

Reply:
This is a basic description at this point. Should that be removed and a note to see the more fully detailed section be added?